LOCALIZATION.md

```diff
@@ -44,39 +70,61 @@ Find existing `[languages]` block in `config.toml` and add a block for the new l
 
-With `i18n/<localization>.toml`, we can setup language-specific site strings.
+`i18n/<localization>.toml` sets up language-specific site strings.
 For a new localization, add a file `i18n/<localization>.toml` based on `i18n/en.toml`.
 `other = "<English site strings>"` in `i18n/<localization>.toml` can be translated.
 
-### Add a new localization directory
+#### 2-3. Adding a new localization directory
 
-Translations of glossary terms should go in the appropriate language directory inside of `/content/`.
+Translated Glossary terms are saved in the appropriate language directory inside `/content/`.
 
-Add a subdirectory in two-letter language code to `content` (`content/<localization>`). 
+Create a subdirectory in `content` and name it using the appropriate two-letter language code (`content/<localization>`).
 For example, 
 ```
 contents/en
 contents/ko
 ...
 ```
 
-### Localize and add minimum required contents
+#### 2-4. Minimum required content
+
+Localize content and save to `content/<localization>`.
+
+Your first PR should include these pages: 
 
-Localize contents and add the files to `content/<localization>`.
+ - Home: `content/en/_index.md` -> `content/<localization>/_index.md`
+ - How to contribute: `content/en/contribute/_index.md` -> `content/<localization>/contribute/_index.md`
+ - Style guide: `content/en/style-guide/_index.md` -> `content/<localization>/style-guide/_index.md`
 
-Following list is for minimum required contents to make a new localization to go live with the website.
+#### 2-5. Check configuration works and open a PR
+
+Before opening a PR, ensure the website with the updated configuration works by running [Hugo server](https://github.com/cncf/glossary#setting-up-a-local-instance). If it does, select the new language on the local website.
+
+![image](https://user-images.githubusercontent.com/5966944/150203331-c535a95c-c36e-4c01-85a8-5261ce6f70de.png)
+
+Open a PR (ex: https://github.com/cncf/glossary/pull/291) and wait for it to be reviewed by the [maintainers](https://github.com/cncf/glossary/blob/main/CODEOWNERS#L6).
+
+### 3. Localizing and adding minimum required terms
+
+Before going live with a localized Glossary, you need at least:
 
- - Home: `content/en/_index.md`
- - How to contribute: `content/en/contribute/_index.md`
- - Style guide: `content/en/style-guide/_index.md`
  - 10 translated terms
 
-### Merge the localization branch into `main` branch
+How localization teams achieve that, is up to them. For instance, they can use issues to assign contributors to translation terms and track progress (ex: https://github.com/cncf/glossary/issues/269).
+
+### 4. Merging the localization branch into `main` branch
+
+After completing the minimum required terms for a new localization, open a PR to merge commits in the localization branch into the `main` branch. (ex: `dev-ko` to `main`).
+
+Once the PR is merged, the localized content will go live on its website 🎉
+
+---
+
+# Join an existing localization team
```

Comment:
Updated this section. I don't think we need a lot more. If there is a team already, they should be able to onboard the newbie, what do you think?

Reply:
Looks good to me :)